rocAL/rocAL_pybind/examples/tf_petsTrainingExample/download_and_preprocess_dataset.sh

```diff
@@ -14,8 +13,10 @@ tar xzvf images.tar.gz
 tar xzvf annotations.tar.gz
 mkdir tf_pets_records
 git clone https://github.com/swetha097/rocALmodels.git
```

Comment:
@swetha097  Where is this repo originally coming from? We might need to add it into MIVisionX-data or someplace common if no copyright issues

Reply:
I will check on this & get back to you Lakshmi

Reply:
Few files like https://github.com/swetha097/rocALmodels/blob/master/models/research/configs/ssd320_bench.config seems to have Nvidia copyright Lakshmi.
Few of them seem to have tensorflow copyright too. 

This codebase seems to originally come from tensorflow & has been modified to run on Nvidia.

Confirmed with Abishek on this as I had forked from his repository.

Reply:
@kiritigowda How do we want to host this model for training example?